modules/pinger/src/main/java/org/hawkular/component/pinger/Log.java

```diff
@@ -27,14 +28,14 @@
  * @author Heiko W. Rupp
  */
 @MessageLogger(projectCode = "HAWKULAR")
-public interface Log {
+public interface Log extends BasicLogger {
```

Comment:
It isn't (http://docs.jboss.org/seam/3/3.1.0.Final/reference/en-US/html/solder-logging.html#jbosslogging), but if you do that, you'll get the generic methods like debug, trace, logf, etc.

If we want to be super strict and allow only the predefined (and localized) methods than it can go away. Btw. alert does it the same way.